docs/Modules/ecdsa.mdx

```diff
@@ -38,6 +38,49 @@ One of the most popular modules, it bridges the ease of traditional web logins w
 2. **dApp Interaction**: Simplifies the process of logging into dApps using Ethereum messages thanks to EIP-1271 support.
 3. **Ownership Management**: Mirrors traditional ownership systems, providing a familiar framework within a more advanced blockchain setting.
 
+## SDK Guide 📘
+
+### Initializing ECDSA Module 🌟
+
+There is no need to do anything in order to use ECDSAOwnershipValidationModule, this module is set by default on every smart account.
+
+### Usage
+
+| Key                                                                                                              | Description                                                                                                                           |
+| ---------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| [signer](https://bcnmy.github.io/biconomy-client-sdk/packages/account/docs/interfaces/SmartAccountSigner.html)   | This signer will be used for signing userOps for any transactions you build. Will accept ethers.JsonRpcSigner as well as a viemWallet |
```

Comment:
these links are not accessible.

Reply:
They will be once this is merged: https://github.com/bcnmy/biconomy-client-sdk/pull/418